README.rst

```diff
@@ -170,6 +173,31 @@ excluded (``blacklist``).
 The actual fields to include or exclude. Field name matching is case
 insensitive and underscores in the beginning of the fields are trimmed.
 
+Unit log levels configuration
+=============================
+
+Unit log levels can be used to specify which log levels you want to set on a per unit basis. Matching supports glob
+patterns. For example, to only process messsages for a systemd-unit called ``test-unit`` with severity ``WARNING`` or higher,
+your config could look like this::
+
+  {
+      "unit_log_levels": {
+          "log_level_name": [
```

Comment:
```suggestion
      "unit_log_levels": [
```

(Toplevel README already said it was a list)

Reply:
I will fix top level README instead as I think this is how it should be. I made it like this to match the behaviour of `field_filters` where you name the `field_filter` so that later in the `reader` section you speficy the name of the `field_filter` you want to use, so that you can use different field filters for different readers.